modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -771,10 +783,16 @@ function render_current_filters( $active_buckets, $instance ) {
 	/**
 	 * Renders a single filter that can be applied to the current search.
 	 *
+	 * We intentionally use a paragraph tag for the heading here to get the heading
+	 * to look similar to the rest of the text. See:
+	 * https://github.com/Automattic/jetpack/issues/8553#issuecomment-358740586
+	 *
 	 * @param array $filter
 	 */
 	function render_filter( $filter ) { ?>
-		<h4  class="widget-title"><?php echo esc_html( $filter['name'] ); ?></h4>
+		<p class="jetpack-search-filters-widget__sub-heading">
```

Comment:
I think this visually looks good, but I think these should remain `h4` elements. We can reset their styles (mostly) by doing something like

```
font-size: inherit;
font-weight: bold;
margin: 0;
padding: 0;
```